### DISCUSSION TITLE
Lab3 fir_tb config write address problem

### DISCUSSION THREAD

**Discussion by liyuan-chang** [bol-edu/HLS-SOC-Discussions#91] [General]
我在使用 makefile 跑 tb 的時候，發現 tb 在寫 tap coefficient 時給的 address 好像是錯的
因為 coefficient 的寬度是 4 bytes，所以在 tb 第 232, 238 行的 write/read address 是不是應該要改成 (12'h20 + 4*k) ？

![fir-tb](https://github.com/bol-edu/HLS-SOC-Discussions/assets/78146388/f5bba38f-34a9-4518-b196-13aadf3e4d8c)

**Comment by ZheChen-Bill** [comment]
根據workbook所定義的address map，確實應該修正為4*k，目前已修正了。
